man/systemd.slice.xml

```diff
@@ -65,7 +65,7 @@
     </para>
 
     <para>Note that slice units cannot be templated, nor is possible to add multiple names to a slice unit by creating
-    additional symlinks to it.</para>
+    additional symlinks to its unit file.</para>
```

Comment:
Well, I dig what you're saying, but to nitpick, it's also possible to create a symlink to the unit file name, without the unit file being there ;)